content/en/metrics/summary.md

```diff
@@ -42,18 +42,20 @@ The search bars provide the most comprehensive set of actions to filter the list
 {{< img src="metrics/summary/facets4.png" alt="Metrics Facet Panel" style="width:75%;">}}
 
 ## Configuration of multiple metrics 
-There are two buttons that allow you to configure multiple metrics at a time: 
 
-{{< img src="metrics/summary/configurationbuttons2.png" alt="Bulk Configuration Buttons" style="width:75%;">}}
+Clicking on **Configure Metrics** gives you three options that you can use to configure multiple metrics at a time: 
 
-* **Calculate Percentiles**: Add percentile aggregations to multiple distribution metrics.
+{{< img src="metrics/summary/configurationbuttons10-11-2024.png" alt="Bulk Configuration Buttons" style="width:75%;">}}
```

Comment:
This image is strangely much smaller than the GIF of manage tags right above it. Can we make it larger? as it's hard to see the text without clicking into the image?

Reply:
Thanks for that feedback, I added a consistent width for all `img` resources in this section in [342324b](https://github.com/DataDog/documentation/pull/25714/commits/342324b76936f11374653f083fc49106417cad56)